app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable, which means it is shared across all instances of `Animal`. This can lead to unintended behavior if not managed carefully, especially in a multi-threaded environment. Consider using an instance variable or a different approach to track alive animals.